docs/developing.rst

```diff
@@ -0,0 +1,199 @@
+Let's suppose that suddenly, you realize that what would be even more useful is a function that
+takes an argument, ``place``, so that the output filename is actually ``hello_PLACE``, with
+``PLACE`` the value passed and ``'world'`` as the default value.
+We merge any changes from the upstream master branch into our branch via
+
+.. code-block:: bash
+
+    git checkout feature/say_hello  # unless you're already there
+    git fetch upstream master
+    git merge upstream/master
+
+and then begin work on our test.
+We need to our unit test to be more complete, so we update it to look more like the following,
+adding several cases to make sure our function is robust to the name supplied:
+
+.. code-block:: python
+
+    def test_say_hello():
+        # prefix of all files to be checked
+        prefix = 'hello_'
+        # suffix of all files to be checked
+        suffix  = '.txt'
+        # run the function with several cases
+        for x in ['world', 'solar system', 'galaxy', 'universe']:
+            # current test name
+            outname = prefix + x + suffix
+            # call the function
+            say_hello(x)
+            # test the function
+            assert op.exists(outname)
+            # clean up from this call
+            os.remove(outname)
+
+Once that test is passing, we may need to adjust the integration test.
+Our program creates a file, ``hello_world.txt``, which the older version would not have produced.
+Therefore, we need to add the file to ``$TEDANADIR/tedana/tests/data/tedana_outputs.txt`` and its
+counterpart, R2-D2-- uh, we mean, ``tedana_outputs_verbose.txt``.
+With that edit complete, we can run the full ``pytest`` suite via
+
+.. code-block:: bash
+
+    pytest $TEDANADIR/tedana/tests
+
+Once that filename is added, all of the tests should be passing and we should open a PR to have our
+change reviewed.
+
+From here, others working on the project may request changes and we'll have to make sure that our
+tests are kept up to date with any changes made as we did before updating the unit test.
+For example, if a new parameter is added, ``greeting``, with a default of ``hello``, we'll need to
+adjust the unit test.
+However, since this doesn't change the typical workflow of ``tedana``, there's no need to change
+the integration test; we're still matching the original filename.
+Once we are happy with the changes and some members of ``tedana`` have approved the changes, our
+changes will be merged!
+
+We should then do the following cleanup with our git repository:
+
+.. code-block:: bash
+
+    git checkout master
+    git fetch upstream master
+    git merge upstream/master
+    git branch -d feature/say_hello
+    git push --delete origin feature/say_hello
+
+and we're good to go!
+
+Monthly Developer Calls
+=======================
+We run monthly developer calls via Zoom.
+You can see the schedule via the tedana `google calendar`_.
+
+Everyone is welcome.
+We look forward to meeting you there :hibiscus:
```

Comment:
```suggestion
We look forward to meeting you there!
```

The emoji aren't supported on .rst, unfortunately 😢

Reply:
Even worse, there is no sad hibiscus emoji for me to respond with.